es-bs-base/scss/_popover.scss

```diff
@@ -24,6 +24,7 @@
   border: variables.$popover-border-width solid variables.$popover-border-color;
   @include border-radius.border-radius(variables.$popover-border-radius);
   @include box-shadow.box-shadow(variables.$popover-box-shadow);
```

Comment:
Noticed this:
<img width="445" alt="Screen Shot 2024-05-07 at 11 45 43 AM" src="https://github.com/EnergySage/es-ds/assets/1350363/f1329701-09f0-4780-a279-eb10974e7cf6">

Had a [conversation with Lauren in Slack](https://energysage.slack.com/archives/C02FU6TCDPX/p1715097269775819) about taking the Modal shadow and using it on both the dark and light versions of the Popover and she agreed - we may tweak the shadow later, but can you take the one from Modal in the Figma and put it in the `$popover-box-shadow` variable? You may need to remove the wrapping `box-shadow.box-shadow()` function since that won't print it out since shadows are disabled globally (and should probably stay that way, lest we affect everything).

Then it'll look more like this, which is definitely better for visibility, though maybe design will tweak the color values later on:
<img width="452" alt="Screen Shot 2024-05-07 at 11 52 55 AM" src="https://github.com/EnergySage/es-ds/assets/1350363/81c346c6-e649-4b9c-9a2f-c8a418283e95">